### ISSUE TITLE
Fix assembly assignments parsing

### ISSUE THREAD

**Issue by LogvinovLeon** [federicobond/solidity-parser-antlr#46]
Solidity accepts both options. This parser fails on first one and only accepts the second one.
I know it's minor nit, but would be still good to accept both.

<img width="654" alt="screen shot 2019-02-20 at 2 48 14 pm" src="https://user-images.githubusercontent.com/6204356/53130266-9c5ad400-351e-11e9-8adb-a5c44f7fe30a.png">

**Comment by federicobond** [comment]
> Solidity accepts both options.
<img src="https://user-images.githubusercontent.com/138426/53134304-6dac2000-3555-11e9-9c07-2a6272763d74.png" width="400" height="400" />

**Comment by federicobond** [comment]
@LogvinovLeon good catch! Is this documented behaviour? Or just an oversight on the part of the Solidity team? Is there any mention of this in the issues or docs for ethereum/solidity?

**Comment by LogvinovLeon** [comment]
@federicobond I don't think it's documented. And I think it's a mistake it supports it. Maybe this should be an issue on a solidity repo instead

**Comment by federicobond** [comment]
Yeah, let's open an issue in the Solidity repo and see if they are willing to tighten things up a bit. I don't think many users expect it to work that way.
